docker-compose.yml

```diff
@@ -9,15 +9,7 @@ services:
     ports:
       - "7896:7896"
     environment:
-      ## spring data
       - spring.datasource.url=jdbc:mysql://mysqldb:3306/lpvs
-      - spring.datasource.username=root
-      - spring.datasource.password=lpvsroot
```

Comment:
These variables have not been implemented in lpvs yet. Right?
We can add these lines to docker-compose.yml in the near future when lpvs read them from env variables.

Reply:
> These variables have not been implemented in lpvs yet. Right? We can add these lines to docker-compose.yml in the near future when lpvs read them from env variables.

Please see comment below from @o-kopysov  -
all features kept- we can use these values from application.properties file.
And it reflected in manual for Docker.
Anyway it also can be returned to previous state if needed.